oaknational/Oak-Web-Application
# fix: update subject phase picker options

## Description
- Integrate new subject phase options MV
- Show only options with published data
- Show all relevant exam boards for secondary science

## How to test

1. Go to https://deploy-preview-1932--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Click on `Subject`
3. You should see subjects with published data (note: Music not currently showing)
4. Click on `Science`
5. The Phase / Examboard section should open
6. Click `Secondary`
7. You should see multiple examboard choices

## Screenshots

How it used to look (delete if n/a):
<img width="993" alt="Screenshot 2023-09-26 at 11 29 09" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/f7103395-28a4-4d9b-b857-92f699d8938d">
<img width="993" alt="Screenshot 2023-09-26 at 11 28 55" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/1209d24e-c071-4443-91a3-d32cd8de06f6">

How it should now look:
<img width="993" alt="Screenshot 2023-09-26 at 11 28 13" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/66e06eb0-336f-444d-a2e4-72dab5916bdf">
<img width="993" alt="Screenshot 2023-09-26 at 11 27 55" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/4b45c4b8-9b8e-428e-b061-6d59d7766d4e">

## Checklist
